packages/suite/src/views/settings/SettingsDevice/DisplayRotation.tsx

```diff
@@ -21,12 +22,20 @@ interface DisplayRotationProps {
 export const DisplayRotation = ({ isDeviceLocked }: DisplayRotationProps) => {
     const dispatch = useDispatch();
     const { device } = useDevice();
-    const currentRotation = device?.features?.display_rotation;
 
-    if (typeof currentRotation !== 'number') {
+    // features.display_rotation cannot be used to determine support because can be defined for devices not supporting rotation (e.g. T2B1).
+    const isSupported =
+        device?.features != undefined &&
```

Comment:
```suggestion
        device?.features != undefined &&
```
Can it also be `null` that you use `!=`? or is it just mistype?

Reply:
No, this is intentionally explicit so that @peter-sanderson doesn't bully me for this.

Reply:
Shall it be `!==`? (one `=` is probably missing.

Reply:
👀 82be73c648e4cf9ef7bdafb8b16c7d82c9ca4d74